package.json

```diff
@@ -13,9 +13,9 @@
   "type": "commonjs",
   "types": "lib/index.d.ts",
   "scripts": {
-    "test": "jest --config=jest.config.json",
+    "test": "node --test",
     "test:ci": "npm run lint && npm run test:typescript && npm run test:coverage",
-    "test:coverage": "jest --config=jest.config.json --coverage",
+    "test:coverage": "c8 --100 npm run test",
```

Comment:
I think you can delete this after https://github.com/fastify/fastify-awilix/pull/134/files#r1573219656